nx.json

```
@@ -39,10 +39,19 @@
       "style": "scss"
     }
   },
-  "defaultProject": "lumberjack-app",
+  "defaultProject": "example",
```

issue: The calculated (implicit) project name is `examples-lumberjack-app`.
```suggestion
  "defaultProject": "examples-lumberjack-app",
```